emily/handler/src/common/error.rs

```diff
@@ -247,6 +248,16 @@ impl From<serde_dynamo::Error> for Error {
         Error::Debug(format!("serde_dynamo::Error - {err:?}"))
     }
 }
+impl From<FromHexError> for Error {
```

Comment:
Not sure if this becomes irrelevant if you make the `ScriptBuf::from_hex(..)` change @matteojug suggested